src/v3/test/integration/__snapshots__/authenticator-email-verification-data.test.tsx.snap

```diff
@@ -70,59 +70,63 @@ exports[`authenticator-email-verification-data should render form 1`] = `
             <div
               class="MuiBox-root emotion-8"
             >
-              <div
-                class="identifier-container MuiBox-root emotion-9"
+              <form
+                class="o-form MuiBox-root emotion-9"
+                data-se="o-form"
+                novalidate=""
```

Comment:
Hmm, any idea how this form is being added? Was this a miss and it should have always been like this?

Reply:
Actually, I see why the form is present now, its because the identifier container is now a child of the `<Form` instead of being a sibling. I think that is ok.

Reply:
Yeah just realized that I came to this conclusion as well on Friday but forgot about the justification